packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -269,3 +276,73 @@ export const NftWidget: React.FC<NftWidgetProps> = ({ ownerHandle, isOwner, nftS
     </Container>
   )
 }
+
+type HookReturn = NftWidgetProps & { shouldRenderWidget: boolean }
+export const useNftWidget = (videoId?: string): HookReturn => {
```

Comment:
```suggestion
type UseNftWidgetReturn = NftWidgetProps & { shouldRenderWidget: boolean }
export const useNftWidget = (videoId?: string): UseNftWidgetReturn => {
```
wdyt? I think it's better, this is analogous to how we name props - we don't name our props types just `Props` or `ComponentProps`, we naming them as `<ComponentNameHere>Props`